### ISSUE TITLE
Introduce network bandwidth quotas in ooniprobe-ios

### ISSUE THREAD

**Issue by anadahz** [ooni/probe#823]
Allow the user to set a specific network bandwidth; the maximum allowed network traffic used to perform tests and submit reports to the collector(s). 

The quota plan could be:
`BandwidthQuota`: On/Off
`BandwidthQuotaMax`: value in MB/GB
`BandwidthQuotaPeriod`: Daily, Weekly, Monthly

Upon quota completion further tests and reporting will not be performed unless the user alters quota options (BandwidthQuota: off) or increases the limit in `BandwidthQuotaMax`.

**Comment by elioqoshi** [comment]
Is this something we want to tackle in the next versions?

**Comment by elioqoshi** [comment]
This would also need some UX work to be implemented

**Comment by hellais** [comment]
I think this is best put into 2.2. The specifications and requirements around this feature are still a bit murky and I suspect we want to fix them only once orchestration is fully rolled out.

**Comment by elioqoshi** [comment]
I was under the impression that Orchestra would require more time than the time 2.2 could arrive by.

In any case, moving this to 2.2 so we can do a bit of housekeeping and keep an eye on it once 2.1 is out. 

Assigning myself for the UX component.

**Comment by lorenzoPrimi** [comment]
I don't think orchestra will be ready for 2.2 either so will probably be postponed to the orchestra release

**Comment by hellais** [comment]
This is also related to: https://github.com/ooni/probe/issues/916

**Comment by hellais** [comment]
At this stage I think this is more of a product/UX question.

I will put in some time in the upcoming sprints to think about this.
